### PR TITLE
Validations for post a job form

### PR DESCRIPTION
**Pull request by vadlachary2002**
1.Removed about us page
2.Added validations to post a job form
updated vaidations
image
image

### CODE REVIEW

File: src/pages/PostJobs/Form.tsx [hunk 1]
```diff
@@ -79,9 +86,17 @@ const Form = () => {
       qualifications:qualifications.map((qualification)=>{return qualification.value;}),
       duties:duties.map((duty)=>{return duty.value;}),
     };
-    setPreview('form hide');
-    showPreview(jobView);
+
+    if(validate(jobView)){
+      console.log('okay');
```
Comment (maharjunm) [inline src/pages/PostJobs/Form.tsx]: remove console.

File: src/pages/PostJobs/validate.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import  {JobDet}  from '../../components/DataModels/JobDet';
+function isDateEmpty(date: Date): boolean {
+  return !date || isNaN(date.valueOf());
+}
+const Validate=(jobView:JobDet)=>{
```
Comment (maharjunm) [inline src/pages/PostJobs/validate.tsx]: method name should be camelCase

File: src/pages/PostJobs/validate.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const {title,qualification,experience}= jobView.job;
+  const {name,type,logo}= jobView.company;
+  const {city,state,country,region}= jobView.location;
+  const {postingDate,expiryDate,closingDate,removingJobDate}= jobView.dates;
+  const salary=jobView.salary;
+  const qualifications =jobView.qualifications;
+  const duties= jobView.duties;
+
+
+  if(title.trim()==='' || qualification.trim()==='' || experience.trim()===''){
```
Comment (maharjunm) [inline src/pages/PostJobs/validate.tsx]: you can just replace this `title.trim() === ''` to `!title`. that should be fine mostly.

**Comment by vadlachary2002** [comment]
Home page dafault logo screenshot
image

**Review by maharjunm — APPROVED** [review]

**Pull request merged**